modules/scheduling/src/main/java/org/projectodd/wunderboss/scheduling/Scheduling.java

```diff
@@ -35,6 +35,7 @@
         public static final ScheduleOption LIMIT     = opt("limit", ScheduleOption.class);
         public static final ScheduleOption UNTIL     = opt("until", ScheduleOption.class);
         public static final ScheduleOption SINGLETON = opt("singleton", true, ScheduleOption.class);
+        public static final ScheduleOption ALLOW_CONCURRENT_EXEC = opt("allow-concurrent-exec?", true, ScheduleOption.class);
```

Comment:
The option name needs to be `"allow_concurrent_exec"` so we can use the option from TorqueBox. Immutant will automatically convert `:allow-concurrent-exec?` to `"allow_concurrent_exec"`.